front/src/router/index.js

```diff
@@ -1,6 +1,7 @@
+import { Cartoons, FfCelebrities, Movies, News, Serials } from '@views';
+
```

Comment:
why do you change it?

Reply:
i dont change it, linter or formater make this.